package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "hubot-ibmcloud-cloudant",
+  "version": "0.0.1",
+  "description": "hubot-ibmcloud-cloudant provides Bluemix Cloudant management functionality.",
+  "main": "index.js",
+  "respository": {
+    "url": "https://github.com/ibm-cloud-solutions/hubot-ibmcloud-cloudant",
+    "type": "git"
+  },
+  "scripts": {
+    "start": "source config/env && hubot -a shell",
+    "slack": "source config/env && hubot -a slack",
+    "facebook": "source config/env && hubot -a fb",
+    "postinstall": "initDb src/nlc/NLC.json",
+    "lint": "eslint src/**/*.js test/*.js index.js",
+    "test": ". test/.env && mocha test",
+    "coverage": ". test/.env && istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
+  },
+  "keywords": [
+    "hubot",
+    "hubot-scripts",
+    "bluemix",
+    "cloud",
+    "slack",
+    "cloudant"
```

Comment:
we have been including these keywords as well in other repos:  "ibm","ibm cloud"